oat-sa/extension-tao-testqti-previewer
# Feature/ms 447/scorer highlight plugin

# Pull request summary

Related to [MS-447](https://oat-sa.atlassian.net/browse/MS-447).

Scorer's Highlight plugin

# How to test

On MS side:

-   Login as a scorer
-   Go to the task page
-   Click on the highlighter icon button
-   Select some text, click on the color button - text should be highlighted
-   Click on the eraser button - highlights should be cleared
